lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -23,11 +25,22 @@ class TestResultsPush:
     URL = "http://pbench.example.com/api/v1"
 
     @staticmethod
-    def add_http_mock_response(code: HTTPStatus = HTTPStatus.OK):
+    def add_http_mock_response(
+        status_code: HTTPStatus = None, message: Optional[Union[str, Dict]] = None
+    ):
```

Comment:
You've changed the default value for the first parameter (replacing `OK` with `None`), but you didn't modify the calls at (new) lines 104, 146, 168, 269, 281, or 294 which depend on it...that seems curious.  What does the status code default to if it is unspecified?  (The [docs](https://github.com/getsentry/responses/blob/master/README.rst#id11) don't seem to say.)

Some of the test scenarios are checking CLI error handling, which should never result in actually making requests, so a bad default shouldn't affect them.  However, for those scenarios, the call is unnecessary, and actually it's inappropriate -- instead it [should be omitted](https://github.com/getsentry/responses/blob/master/README.rst#id8) such that any request results in an exception.

For the other tests, I think the scenarios are counting on getting a "success" status, so the default should probably be `HTTPStatus.CREATED`, now.  (I'm not sure how these tests are passing, currently....  😞)

Reply:
The status is None, so it does not get added to the response at all. In that case, `responses` fills it in with a 200 by default.